python/semantic_kernel/connectors/memory/pinecone.py

```diff
@@ -41,17 +38,36 @@
 logger = logging.getLogger(__name__)
 
 
-DISTANCE_METRIC_MAP = {
+DISTANCE_METRIC_MAP: Final[dict[DistanceFunction, Metric]] = {
     DistanceFunction.COSINE_SIMILARITY: Metric.COSINE,
     DistanceFunction.EUCLIDEAN_DISTANCE: Metric.EUCLIDEAN,
     DistanceFunction.DOT_PROD: Metric.DOTPRODUCT,
+    DistanceFunction.DEFAULT: Metric.COSINE,
 }
 
 
+class PineconeSettings(KernelBaseSettings):
```

Comment:
Can you help me understand why there's a `PineconeSettings` here as well as in `memory_stores/pinecone/pinecone_settings.py`?